src/pages/settings/Subscription/SubscriptionSize/utils.ts

```diff
@@ -0,0 +1,19 @@
+import {addMonths, format, startOfMonth} from 'date-fns';
+import type {FormInputErrors, FormOnyxValues} from '@components/Form/types';
+import * as ValidationUtils from '@libs/ValidationUtils';
+import CONST from '@src/CONST';
+import type ONYXKEYS from '@src/ONYXKEYS';
+import INPUT_IDS from '@src/types/form/SubscriptionSizeForm';
+
+const validate = (values: FormOnyxValues<typeof ONYXKEYS.FORMS.SUBSCRIPTION_SIZE_FORM>): FormInputErrors<typeof ONYXKEYS.FORMS.SUBSCRIPTION_SIZE_FORM> => {
+    const errors = ValidationUtils.getFieldRequiredErrors(values, [INPUT_IDS.SUBSCRIPTION_SIZE]);
+    if (values[INPUT_IDS.SUBSCRIPTION_SIZE] && !ValidationUtils.isNumeric(values[INPUT_IDS.SUBSCRIPTION_SIZE])) {
```

Comment:
We should also disallow `0` - not sure if you'd like to handle here or if it'll be covered in a future PR

Reply:
I will change it now